test/functional/ui/highlight_spec.lua

```diff
@@ -194,3 +194,60 @@ describe('Default highlight groups', function()
     feed('<cr>') --  skip the "Press ENTER..." state or tests will hang
   end)
 end)
+
+describe('New special highlight color attribute', function() 
+  local screen
+
+  before_each(function()
+    clear()
+    screen = Screen.new(25,10)
+    screen:attach()
+    screen:set_default_attr_ignore({
+      [1] = {bold=true}, 
```

Comment:
please remove the bold ignore (we only ignore the bold-blue group per policy as it would clutter every visual test with the ~-marks). Also for `attr_ignore` there is no point writing out the indices, as these are never referred.